src/lib.rs

```diff
@@ -197,7 +197,7 @@ pub mod pallet {
 		#[pallet::weight(10_000)]
 		/// Deposits funds for a participant into a channel.
 		///
-		/// The `funding_id` is calculated with [Pallet::calc_funding_id].
+		/// The `funding_id` is calculated with [Funding::id].
```

Comment:
Isn't it `Funding.id`?

Reply:
The signature of the function is `Funding::id ` since it is a member function.  
`Funding.id` would be a member function access, not a type.  
The CI checks for all doc references so you dont need to worry about them being invalid :wink: .